plugins/include/message_const.inc

```diff
@@ -799,6 +861,19 @@ enum
  */
 #define TE_BREAKMODEL               108
 
+/**
+ * Flags used in te_create_break_model()
+ */
+#define BreakModel_TypeMask                  0x4F
```

Comment:
Caps?

Reply:
@WPMGPRoSToTeMa IMO it looks better this way, I'll change it to const or enum though.

Reply:
@OciXCrom tagged enum should fit good here, tagged constants should be good too.

Reply:
@WPMGPRoSToTeMa I changed it to `enum`, but I'm not sure if I can use (<< 1) in the tag for HEX values. I assume you know?

Reply:
@OciXCrom hex doesn't make difference, of course you can use `(<< 1)`.

Reply:
@WPMGPRoSToTeMa What about `0x4F`? It outputs 79.

Reply:
@OciXCrom you can define it as:
```Pawn
BreakModel_TypeMask = BreakModel_Glass | BreakModel_Metal | BreakModel_Flesh | BreakModel_Wood | BreakModel_Concrete
```
or:
```Pawn
const BreakModelFlags:BreakModel_TypeMask = BreakModel_Glass | BreakModel_Metal | BreakModel_Flesh | BreakModel_Wood | BreakModel_Concrete;
```

Reply:
@WPMGPRoSToTeMa Thanks, I added the first one in the end of the `enum`.